### DISCUSSION TITLE
Didn't work well with `<C-R>=escape(@",'/\')<CR>`

### DISCUSSION THREAD

**Discussion by dakennguyen** [folke/flash.nvim#62] [Q&A]
Hi all,

Basically, I want to use `/\V` with the sentence in my register but I want to escape some characters, so I have this mapping in my config:
```lua
map('v', '*', [[y/\V<C-R>=escape(@",'/\')<CR><CR>]])
```

However, it does not work anymore and I can only work with this config:
```lua
map('v', '*', [[y/\V<C-R>"<CR>]])
```

I would appreciate your help 🙏

**Comment by folke** [comment]
Can you be more explicit in what is not working anymore? Flash only enhances search, it doesn't really replace search.

**Comment by dakennguyen** [comment]
ah sorry, this is my minimal config
```lua
local lazypath = vim.fn.stdpath("data") .. "/lazy/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({
    "git",
    "clone",
    "--filter=blob:none",
    "https://github.com/folke/lazy.nvim.git",
    "--branch=stable", -- latest stable release
    lazypath,
  })
end
vim.opt.rtp:prepend(lazypath)

require("lazy").setup({
  {
    "folke/flash.nvim",
    event = "VeryLazy",
    keys = {
      {
        "s",
        mode = { "n", "o", "x" },
        function()
          require("flash").jump()
        end,
      },
      {
        "S",
        mode = { "n", "o", "x" },
        function()
          require("flash").treesitter()
        end,
      },
    },
    opts = {
      jump = {
        autojump = true,
      },
      highlight = {
        groups = {
          label = "HopNextKey",
        },
      },
      modes = {
        search = {
          enabled = true,
        },
        char = {
          highlight = { backdrop = false },
        }
      }
    },
    config = true,
  }
}, {})

vim.api.nvim_set_keymap('v', '*', [[y/\V<C-R>=escape(@",'/\')<CR><CR>]], { noremap = true })
```

When I disable search mode, and use `*` in visual mode:

https://github.com/folke/flash.nvim/assets/18320285/3d5d017c-d4d1-4ecd-90a7-652ad74fe519

When I enable it:

https://github.com/folke/flash.nvim/assets/18320285/e987652f-98cc-42c3-b43d-3382328c285e

**Comment by folke** [comment]
Can't make it work without flash:

```lua
vim.keymap.set("v", "*", [[y/\V<C-R>=escape(@",'/\')<cr><cr>]], { remap = false })
```

And this test file:

```vim
let s:save_splitright = &splitright
let s:save_splitright = &splitright
let s:save_splitright = &splitright
let s:save_splitright = &splitright
let s:save_splitright = &splitright
let s:save_splitright = &splitright
let s:save_splitright = &splitright
let s:save_splitright = &splitright
```

I'm on the first line, press `V` and then `*`. I get `E486: Pattern not found: \Vlet s:save_splitright = &splitright^@`

**Reply by dakennguyen** [reply]
@folke hi, instead of `V`, would it work if you press `v` and select the sentence?

**Reply by folke** [reply]
That does works indeed. But it also works fine with flash enabled for me:
![image](https://github.com/folke/flash.nvim/assets/292349/ce1bf650-ba42-4cf1-b1c6-fbd0f016cfc2)

**Reply by dakennguyen** [reply]
Weirdly enough for that file, sometimes it works and sometimes it doesn't for me.
Could you try to create `test.py` in the same directory
```python
from fastapi import APIRouter, Depends
from datetime import datetime, timezone
```
select `fastapi` in visual mode and press `*`. 
Then go to the previous file and select `let s:save_splitright = &splitright` and press `*`. Could you see the error now?

**Reply by folke** [reply]
Nope, still works fine for me with flash enabled

**Reply by dakennguyen** [reply]
ah I restart the terminal and now I think it works for me with the minimal config too. However, it still doesn't work with my full config. I'm gonna try to find which plugin caused the conflict. Thank you for your help!
